src/main/java/com/getindata/connectors/http/internal/utils/JavaNetHttpClientFactory.java

```diff
@@ -88,6 +91,21 @@ private static SSLContext getSslContext(Properties properties) {
         String clientPrivateKey = properties
             .getProperty(HttpConnectorConfigConstants.CLIENT_PRIVATE_KEY, "");
 
+        if (StringUtils.isNullOrWhitespaceOnly(keyStorePath)
+            && !selfSignedCert
+            && serverTrustedCerts.length == 0
+            && StringUtils.isNullOrWhitespaceOnly(clientCert)
+            && StringUtils.isNullOrWhitespaceOnly(clientPrivateKey)) {
```

Comment:
What happens if we only have client certs. In this case, we want the trusted public server certs and the client certs. The current fix does not deal with that combination. Can we get hold of the default public server certs and add them like  `securityContext.addCertToTrustStore(cert);  ` then process the client certs?